README.md

```diff
@@ -55,13 +56,19 @@
 - [Gift Card NextJS](https://github.com/adalicious/aiken-gift-card) - A similar gift card example but with NextJS
+- [Gift Card MeshJS](https://github.com/MeshJS/mesh/tree/main/packages/mesh-contract/src/giftcard) - Gift card contract with full integration with MeshJS, offering code example and explanation.
+- [Escrow MeshJS](https://github.com/MeshJS/mesh/tree/main/packages/mesh-contract/src/escrow) - Escrow contract with full integration with MeshJS, offering code example and explanation.
+- [Marketplace MeshJS](https://github.com/MeshJS/mesh/tree/main/packages/mesh-contract/src/marketplace) - Cardano native token marketplace contract with full integration with MeshJS, offering code example and explanation.
+- [Payment Splitter MeshJS](https://github.com/MeshJS/mesh/tree/main/packages/mesh-contract/src/coupon-bond-guaranteed) - Payment splitter contract with full integration with MeshJS, offering code example and explanation.
+- [Simple Swap MeshJS](https://github.com/MeshJS/mesh/tree/main/packages/mesh-contract/src/swap) - Simple swap contract for token exchange with full integration with MeshJS, offering code example and explanation.
+- [Vesting Contract MeshJS](https://github.com/MeshJS/mesh/tree/main/packages/mesh-contract/src/vesting) - Vesting contract for token exchange with full integration with MeshJS, offering code example and explanation.
 - [Maestro Dapps Tutorials](https://github.com/maestro-org/dapp-platform-tutorials/tree/main/bets)
 - [Logical Mechanism's Distributed Representation](https://github.com/logicalmechanism/distributed_representation) - A model semi-liquid `mint-lock-stake` DAO
 - [Logical Mechanism's Convert Expression](https://github.com/logicalmechanism/convert-expression) - A novel variation of a Schnorr protocol
 - [From Aiken to frontend deployment](https://meshjs.dev/guides/aiken) - A end-to-end guide from writing a Aiken smart contract, to creating transactions, to deploying a frontend to interact with it. See also [demo](https://aiken-next-ts-template.vercel.app/) and [repo](https://github.com/MeshJS/aiken-next-ts-template).
 - [Cardano Capture The Flag](https://github.com/vacuumlabs/cardano-ctf) - A game where Cardano developers and enthusiasts can try to exploit purposely vulnerable smart contracts and learn about the most common security issues and how to prevent them.
 - [Cardano Recorded Mint](https://github.com/keyan-m/cardano-recorded-mint) - A proof-of-concept for a minting script with historical record of all mints, and on-chain guarantee of uniqueness.
-- [Gimbalabs Aiken Essentials Module](https://plutuspbl.io/modules/303/slts) - Learn Aiken from scratch and develop your skills with hands-on exercises and mini-projects. 
+- [Gimbalabs Aiken Essentials Module](https://plutuspbl.io/modules/303/slts) - Learn Aiken from scratch and develop your skills with hands-on exercises and mini-projects.
```

Comment:
The website isn't really working for me? None of the tutorial are accessible.

Reply:
I update this line, think just formatter deleting last space. For the link, my best info is that it keep updating, maybe now is in middle of something.

Reply:
Perhaps we should add it later, once it stabilizes?

Reply:
Oh typo, actually i "didnt" update this line. I can pass the message to James for this, Im not in the loop for this so not the person to make decision

Reply:
Ho. I see. My bad 🤦